src/blueprint/server.rs

```diff
@@ -73,21 +66,7 @@ impl TryFrom<crate::config::ConfigModule> for Server {
     fn try_from(config_set: config::ConfigModule) -> Result<Self, Self::Error> {
         let config_server = config_set.server.clone();
 
-        let http_server = match config_server.clone().get_version() {
-            HttpVersion::HTTP2 => {
-                let cert = Valid::from_option(
-                    config_server.cert.clone(),
-                    "Certificate is required for HTTP2".to_string(),
-                );
-                let key = Valid::from_option(
-                    config_server.key.clone(),
-                    "Key is required for HTTP2".to_string(),
-                );
-
-                cert.zip(key).map(|(cert, key)| Http::HTTP2 { cert, key })
-            }
-            _ => Valid::succeed(Http::HTTP1),
-        };
```

Comment:
Why are we removing this validation? I think we should keep this check here and also add unit tests so that we don't break it later.

Reply:
I thought we had that because we needed to pass cert to server, which we do not need anymore, so I removed